src/aks-preview/azext_aks_preview/decorator.py

```diff
@@ -1694,15 +1696,30 @@ def get_disk_driver(self) -> Optional[ManagedClusterStorageProfileDiskCSIDriver]
                 "--disable-disk-driver at the same time."
             )
 
+        if not enable_disk_driver and azuredisk_csi_version:
+            raise ArgumentUsageError(
+                "Cannot specify --azure-csi-version without "
+                "--enable-disk-driver at the same time.")
```

Comment:
Recommend: "The parameter --enable-disk-driver is required when --azure-csi-version is specified."